### DISCUSSION TITLE
Using ScaleData multiple times in HTO vignette

### DISCUSSION THREAD

**Discussion by hannasj** [satijalab/seurat#5046] [Q&A] [answered]
I'm trying to follow the hashing vignette https://satijalab.org/seurat/articles/hashing_vignette.html 
First the whole Seurat object is scaled
pbmc.hashtag <- ScaleData(pbmc.hashtag, features = VariableFeatures(pbmc.hashtag))
later, once each cell has been assigned a HTO and doublets and negative have been removed, the data are scaled again
pbmc.singlet <- subset(pbmc.hashtag, idents = "Singlet")
pbmc.singlet <- ScaleData(pbmc.singlet, features = VariableFeatures(pbmc.singlet))
Please could you explain why is this and whether it matters?

**Comment by timoast** [comment, marked as answer]
Answered here: https://github.com/satijalab/seurat/issues/5053
